src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -28,7 +28,7 @@
  *
  * Parameter and service keys are case insensitive.
  *
- * A service id can contain lowercased letters, digits, underscores, and dots.
+ * A service id can contain letters, digits, underscores, and dots.
```

Comment:
This doesn't seem correct anymore. Service ids are not actually validated are they? They can also contain namespace backslashes for example.